.github/pull_request_template.md

```diff
@@ -58,16 +58,16 @@ gitpod:summary
 </details>
 
 <details>
-<summary>Preview Environment</summary>
+<summary>Preview Environment / Integration Tests</summary>
 
 - [ ] /werft with-local-preview
       If enabled this will build `install/preview`
 - [ ] /werft with-preview
 - [ ] /werft with-large-vm
-- [ ] /werft with-gce-vm
+- [x] /werft with-gce-vm
```

Comment:
Do the integration tests work if this is disabled?
What's the state of that option? I'm not up to date so not sure what we'd consider the "stability" level here

Reply:
Not sure, I think we don't really support harvester preview envs anymore, but the possibility is still there? Maybe we can just remove this option and always run on gce

Reply:
I believe they continue to work. For example, I don't think I've ever set the `gce-vm` option and was able to work with previews

Reply:
Let's give this a go and see.

Reply:
ah interesting, it only looks like harvester was disabled for `leeway run dev:preview`: https://github.com/gitpod-io/gitpod/commit/47d1412e9587678722f2eb02a0fa9787fc04e539#diff-d6b3d97197c35d055f9215e9dbfe8f5fa0a04b1581d264842ce004061ef31dfdR37

For the integration tests we do need a gce (large) vm, as it attaches additional disks needed to run many workspaces in the environment, which aren't there in harvester preview envs. It probably makes sense to align the gce option with what's used locally